CHANGELOG.asciidoc

```diff
@@ -174,6 +174,12 @@ https://github.com/elastic/beats/compare/v1.2.0...5.0.0-alpha1[View commits]
 - Add `event_id`, `level`, and `provider` configuration options for filtering events {pull}1218[1218]
 - Add `include_xml` configuration option for including the raw XML with the event {pull}1218[1218]
 
+==== Known issues
+* All Beats can hang or panic on shutdown if the next server in the pipeline (e.g. Elasticsearch or Logstash) is
+  not reachable. {issue}1319[1319]
+* When running the Beats as a service on Windows, you need to manually load the Elasticsearch mapping
+  template. {issue}1315[1315]
```

Comment:
Yeah, that works as well, but I thought a workaround is enough for the "known issues". There's a link to the ticket.